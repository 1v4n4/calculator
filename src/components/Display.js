import React from 'react';

export default class Display extends React.Component {
  constructor(props) {
    super(props);
    this.props = {
      result: '',
    }
  }
  render() {
    return <div className='display'>{this.props.result}</div>
  }
}

Display.defaultProps = {
  result: '0'
};