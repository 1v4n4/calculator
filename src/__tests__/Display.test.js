import renderer from 'react-test-renderer';
import Display from '../components/Display';

it('renders Display', () => {
  const display = renderer.create(<Display />).toJSON();
  expect(display).toMatchSnapshot();
});
