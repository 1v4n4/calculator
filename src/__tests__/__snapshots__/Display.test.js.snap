// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Display 1`] = `
<div
  className="display"
>
  0
</div>
`;
